app/src/main/java/com/jHerscu/clearskies/ui/components/FloatingMenuButton.kt

```diff
@@ -0,0 +1,51 @@
+package com.jHerscu.clearskies.ui.components
+
+import androidx.compose.foundation.Image
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Menu
+import androidx.compose.material3.DrawerState
+import androidx.compose.material3.ExtendedFloatingActionButton
+import androidx.compose.material3.FloatingActionButtonDefaults
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import com.jHerscu.clearskies.R
+import com.jHerscu.clearskies.ui.theme.CARD_TONAL_ELEVATION_DP
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
+
+@Composable
+fun FloatingMenuButton(
```

Comment:
Re-isolate. Rm from row in pref screen and move carousel & switch into scrolling column

Reply:
Moving carousel down, but leaving Group by column in top